tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -196,6 +213,8 @@ test.describe( 'Set up accounts', () => {
 				baseURL + 'google_auth'
 			);
 
+			setUpAccountsPage.getTermsCheckbox().check();
```

Comment:
Awaiting the async action would be better.

```suggestion
			await setUpAccountsPage.getTermsCheckbox().check();
```